CHANGELOG.md

```diff
@@ -402,7 +403,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Allow option to skip device selection page in demo app.
 - Add demo hook for debugging media connections
-- Add github link to getNearestMediaRegion method in README.md
```

Comment:
Let's not remove this one since it was added back then under 1.6.0 release. Your changelog with just the documentation under [Unreleased] under Fixed section is good.